app/concerns/liquid_interpolatable.rb

```diff
@@ -189,6 +189,11 @@ def uri_expand(url, limit = 5)
       url
     end
 
+    # Rebase URIs contained in attributes in a given HTML fragment
+    def rebase_html(input, base_uri)
```

Comment:
Actually I've had an HTML tag in mind, called `<base>` which has an `href` attribute, so am starting to like `rebase_hrefs` better, because it may be extended to XML or other data formats/structures in future.